src/toolkits/activity_classification/activity_classifier.cpp

```diff
@@ -442,11 +441,8 @@ gl_sframe activity_classifier::predict_per_window(gl_sframe data,
 
   // Accumulate the class probabilities for each prediction window.
   gl_sframe raw_preds_per_window = perform_inference(data_it.get());
-
-  gl_sframe result =
-      gl_sframe({{"session_id", raw_preds_per_window["session_id"]},
-                 {"probability_vector", raw_preds_per_window["preds"]}});
-  result = result.add_row_number("prediction_id");
+  gl_sframe result = gl_sframe({{"session_id", raw_preds_per_window["session_id"]},
+                  {"probability_vector", raw_preds_per_window["preds"]}});
```

Comment:
The formatting here looks strange. Have you run `git clang-format` (just on your changes, please, not the entire file)?

Reply:
I will run it again. Thank you!